subscriptions/tests.py

```diff
@@ -256,6 +256,16 @@ def test_create_subscription_data(self):
         self.assertEqual(d.process_status, 0)
         self.assertEqual(d.metadata["source"], "RapidProVoice")
 
+    def test_check_missing_data_subscription_response(self):
+        # Setup
+        post_subscription = {}
+        # Execute
+        response = self.client.post('/api/v1/subscriptions/',
```

Comment:
Are you testing the right endpoint here? `/api/v1/subscriptions/` uses the `SubscriptionViewSet` class.
I think you want `/api/v1/subscriptions/request/` which uses the `SubscriptionRequest` class. You can see this mapping in the subscriptions/urls.py file (https://github.com/praekelt/seed-stage-based-messaging/blob/develop/subscriptions/urls.py)

The tests for the SubscriptionRequest class are a bit lower down, in the `TestSubscriptionsWebhookListener` class. I think this test belongs there with them?
Otherwise it looks good.